staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -151,6 +151,12 @@ type SharedInformer interface {
 	// because the implementation takes time to do work and there may
 	// be competing load and scheduling noise.
 	AddEventHandlerWithResyncPeriod(handler ResourceEventHandler, resyncPeriod time.Duration)
+	// RemoveEventHandler removes an event handler from the shared informer.
+	// It returns true if the removal is successful
+	RemoveEventHandler(handler ResourceEventHandler) bool
```

Comment:
better to return err, bool

Reply:
I will address in separate PR.

Reply:
Done. See https://github.com/kubernetes/kubernetes/pull/98536/files#diff-10a340e966883e5205df63ad90ba6827f38f8a8d4b8614152c0baae9813a56c6R637